library/std/src/hash/random.rs

```diff
@@ -2,9 +2,6 @@
 //! [`collections`] module without actually publicly exporting them, so that parts of that
 //! implementation can more easily be moved to the [`alloc`] crate.
 //!
-//! Although its items are public and contain stability attributes, they can't actually be accessed
-//! outside this crate.
-//!
```

Comment:
Taking a closer look, I think the comments are actually correct. This module was added in https://github.com/rust-lang/rust/pull/115694 for the reasons the comment mentions. I think we can just clarify this comment by mentioning that the types are re-exported elsewhere.
```diff
- //! Although its items are public and contain stability attributes, they can't actually be accessed
- //! outside this crate.
+ //! Although its items are public and contain stability attributes, they can't actually be accessed
+ //! through this module, and are instead re-exported from `std::hash` and `std::collections::hash_map`.
```